src/header.ts

```diff
@@ -116,7 +116,7 @@ export class WelcomeHeader extends LitElement {
 
     @media only screen and (max-width: 500px) {
       .pic {
-        background-image: url('https://archive.org/download/democracys-library/web-component/banner-image-mini.webp');
+        background-image: url('https://archive.org/download/democracys-library/web-component/banner-image-2.webp');
```

Comment:
This should be banner-image-mini-2.webp (it's still referencing the large image here)